website/docs/r/aws_secret_backend_static_role.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: "vault"
+page_title: "Vault: vault_aws_secret_backend_static_role resource"
+sidebar_current: "docs-vault-resource-aws-secret-backend-static-role"
+description: |-
+  Creates a static role for the AWS secret backend for Vault.
+---
+
+# vault\_aws\_secret\_backend\_static\_role
+
+Creates a static role for AWS Secret Backend for Vault.
+
+~> **Important** All data provided in the resource configuration will be
+written in cleartext to state and plan files generated by Terraform, and
+will appear in the console output when Terraform runs. Protect these
+artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
+
+## Example Usage
+
+```hcl
+
+resource "vault_aws_secret_backend" "aws" {
+	path = "%s"
+	description = "Obtain AWS credentials."
+}
+
+resource "vault_aws_secret_backend_static_role" "role" {
+	backend = vault_aws_secret_backend.aws.path
+	name = "test"
+	username = "%s"
+	rotation_period = "3600"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `namespace` - (Optional) The namespace to provision the resource in.
+  The value should not contain leading or trailing forward slashes.
+  The `namespace` is always relative to the provider's configured [namespace](/docs/providers/vault#namespace).
+  *Available only for Vault Enterprise*.
+
+* `backend` - (Optional) The unique path this backend should be mounted at. Must
+  not begin or end with a `/`. Defaults to `aws`
+
+* `username` - (Required) The username of the existing LDAP entry to manage password rotation for.
```

Comment:
```suggestion
* `username` - (Required) The username of the existing AWS IAM to manage password rotation for.
```

Reply:
🤦🏽‍♀️  I even checked to make sure I didn't miss any